### PR TITLE
fix(sakavottord-application): Minor UI changes

### PR DESCRIPTION
**Pull request by jonbjarnio**
# ...

Attach a link to issue if relevant

## What

Replacing images with SVG and changing the explination of the sealed document

## Why

Images are broken on dev and the explanation is wrong

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by siggifv — CHANGES_REQUESTED** [review]
Is the addition of this file `libs/api/domains/syslumenn/src/lib/client/syslumenn.client.ts` intentional? It's not in line with the PR title and description. Furthermore, this is an outdated version of the Syslumenn client. The Syslumenn client was recently refactored to be generated based on OpenAPI definition. Did you mean to change the following line?
https://github.com/island-is/island.is/blob/main/libs/clients/syslumenn/src/lib/syslumennClient.service.ts#L104

**Comment by jonbjarnio** [comment]
> Is the addition of this file `libs/api/domains/syslumenn/src/lib/client/syslumenn.client.ts` intentional? It's not in line with the PR title and description. Furthermore, this is an outdated version of the Syslumenn client. The Syslumenn client was recently refactored to be generated based on OpenAPI definition. Did you mean to change the following line? https://github.com/island-is/island.is/blob/main/libs/clients/syslumenn/src/lib/syslumennClient.service.ts#L104

I accidentally added it back, updated the PR where I removed the file and updated the correct one. :)

**Review by darrikonn — CHANGES_REQUESTED** [review]
Maybe I'm missing something ... would be good to explain your changes in the PR's description.

Why do you have to render this as a react component?


BTW. the branch has 160 commits :D ... something went wrong.

File: libs/application/templates/criminal-record/src/assets/Bus.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
+import React, { FC } from 'react'
+
+export const Bus: FC = () => (
```
[the remark this answers is not in the document]
Comment (jonbjarnio) [reply libs/application/templates/criminal-record/src/assets/Bus.tsx:3]: It was previously referenced like that but we got 404 error, so I converted it to a component.

[the remark this answers is not in the document]
Comment (darrikonn) [reply libs/application/templates/criminal-record/src/assets/Bus.tsx:3]: I see ... would be good to delete the unused SVGs from assets with these changes.

**Comment by eirikurn** [comment]
Should this PR be closed in favor of #6149?

**Comment by darrikonn** [comment]
This was moved to https://github.com/island-is/island.is/pull/6149

**Pull request closed without merging**